addons/dockable_container/layout.gd

```diff
@@ -26,16 +42,16 @@ func _init() -> void:
 	resource_name = "Layout"
 
 
-func set_root(value: LayoutNode, should_emit_changed = true) -> void:
+func set_root(value, should_emit_changed = true) -> void:
 	if not value:
 		value = LayoutPanel.new()
```

Comment:
On this line the root resource is set to the "Tabs" Resource. Perhaps there is the problem? 
Maybe you can tell me the diffrence between Split and Tabs.
I think Split makes smaller predefined "windows" and Tabs make them all in one big "window" using tabs.
Is this correct?

Reply:
Yeah, by what you wrote, I think you got it right.
The Resource named "Tabs" is an instance of `layout_panel.gd`, which holds actual tabs: their names and which one is the currently active one.
Splits (instances of `layout_split.gd`), on the other hand, are Resources that hold a "split": the separation between two nodes, be them splits or panels. They can be [horizontal or vertical](https://github.com/Gamemap/godot-dockable-container/blob/main/addons/dockable_container/layout_split.gd#L5-L8), and there's the percentage of the width/height where this split handle is.

Now, any of them can be the root of a layout Resource. If you have the plugin with the custom inspector on, you should probably just edit the layout in it and let the plugin setup the correct Resources. If you split the view in the inspector, the Layout shown should change to a Split automatically.
How exactly are you trying to change the layout?

<hr/>

As a sidenote, I don't reeeeally like this nested Resources thing. I though at some point about making the layout just be nested dictionaries instead of nested objects, to have only the Layout Resource, but never worked on refactoring it.

Reply:
> How exactly are you trying to change the layout?

After adding your comments the inspector shows only a gray squared (where you should be able to edit the comtainers).
Sorry, but I forgot to mention that the image above is from Godot 3.4.

----

Godot 4.0 alpha2:
![dockable_container_1](https://user-images.githubusercontent.com/71942164/156857436-8270de8b-2c08-42e6-8dcb-55f76f94b4b9.png)

Reply:
Hm, ok. I you have this gray box, it means the plugin and the layout editor were at least loaded.
There must be some error still going on, since the controls never appeared, and you can check them out in the Output panel.
See the red circle in your "Ausgabe" tab below the scene? It means some error message occurred. I guess it will come from the `layout_editor_property.gd` or the other `layout_*` scripts that are being run by the plugin. Could you open it and see which error is it giving?

Reply:
Thank you for the hint with the "Ausgabe". I found the error, fixed it and it seems to be working. In a few hours I will be able to test it more precisely and will commit it then.

----

layout_editor_property.gd, line 48
```
	new_control.align = Label.ALIGN_CENTER
	new_control.valign = Label.VALIGN_CENTER
 ```
 ```
	new_control.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
	new_control.vertical_alignment = VERTICAL_ALIGNMENT_CENTER
 ```